bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/gridCellEditor.js

```diff
@@ -51,13 +51,16 @@ Ext.define('pimcore.object.helpers.gridCellEditor', {
             return;
         }
 
-        var title = fieldInfo.label ? fieldInfo.label : fieldInfo.key;
-
         this.context = this.editingPlugin.context;
         // this.callParent(arguments);
 
         var tagType = fieldInfo.layout.fieldtype;
 
+        // translate title
+        if(typeof fieldInfo.layout.title != "undefined") {
+            fieldInfo.layout.title = t(fieldInfo.layout.title);
```

Comment:
shouldn't it be ts instead ?

Reply:
`ts()` is only a alias for `t()`
https://github.com/pimcore/pimcore/blob/8b21bdd0e5cbb42b4261e2e596849f57292d8529/bundles/AdminBundle/Resources/public/js/pimcore/functions.js#L64-L66

So i thought `t()` should be preferred